transcom/mymove
# B 21213 int

## [B-21213](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1012916)

## Summary

This task updates the PPM closeout queue to show only Needs Closeout ppms. An issue was found where moves with multiple ppms were displaying incorrectly in the queuemoves table, this was also resolved.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office Application
2. Login as a Service Counselor 
3. Navigate to the Closeout Queue
4. Verify that all moves shown are in the Needs Closeout status

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
